libs/webassembly/Makefile_argon2

```diff
@@ -4,12 +4,15 @@
 TOTAL_MEMORY=48MB
 
 CC=emcc
+WASM2JS=wasm2js
```

Comment:
I don't think wasm2js should be part of the makefiles. This is one example of it: where does wasm2js come from? If we run it in JS then we can just import it

Reply:
wasm2js is inside [`Binaryen`](https://github.com/WebAssembly/binaryen?tab=readme-ov-file) and Binaryen is inside [`Emscripten`](https://github.com/WebAssembly/binaryen/blob/98f08effb18a94618ef12f25134c289836fc316e/README.md?plain=1#L29). So i think it's fine to be part of the make files (?)

Reply:
okay yeah that's a good argument